src/main/java/pages/UtilitiesAka.java

```diff
@@ -0,0 +1,38 @@
+
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import javax.xml.xpath.XPath;
+
+
+public final class UtilitiesAka {
+
+    private static WebDriver driver;
+    private static int timeOut = 10;
+
+    public UtilitiesAka(WebDriver driver) {
+        PageFactory.initElements(driver, this);
+        this.driver = driver;
+    }
+
+
+
+    public String ElementAssertAndFill(String text, String invullen){
+        WebDriverWait wait = new WebDriverWait(driver,timeOut);
+        wait.until(ExpectedConditions.presenceOfElementLocated(By.xpath(text)));
```

Comment:
Je vult hier text in wat prima is. Je assert hier alleen niet het element. Je hebt alleen een wacht conditie en als daar niet aan wordt voldaan krijg je een timeout, geen failed assertion. Omdat het een helper methode is zou ik ook geen assertion hier doen maar noem de methode dan ook geen assertAndFill om verwarring te voorkomen